resources/views/frontend/catalog-new/index-new.blade.php

```diff
@@ -78,19 +77,92 @@ class="tw-absolute tw-left-0 tw-top-36 tw-z-10 tw-w-screen tw-px-16">
                             </filter-show-more>
                         </div>
                     </filter-new-popover-group-controller>
-                    <filter-show-more class="tw-visible tw-pt-4 md:tw-hidden"
-                        :is-extended-open="isExtendedOpen"
-                        :toggle-is-extended-open="toggleIsExtendedOpen">
-                    </filter-show-more>
+                    {{-- Mobile Filter --}}
+                    <filter-disclosure-controller v-slot="fm">
```

Comment:
The `fm` abbreviation doesn't really make sense anymore
```suggestion
                    <filter-disclosure-controller v-slot="dc">
```

```suggestion
                    <filter-disclosure-controller v-slot="fdc">
```